src/main/java/seedu/weme/logic/parser/commandparser/exportcommandparser/ExportCommandParser.java

```diff
@@ -30,9 +31,17 @@ public ExportCommand parse(String args) throws ParseException {
             throw new ParseException(MESSAGE_MISSING_PATH);
         }
 
-        DirectoryPath path = ParserUtil.parseDirectoryPath(argMultimap.getValue(PREFIX_FILEPATH).get());
-
-        return new ExportCommand(path);
+        String directoryPath = argMultimap.getValue(PREFIX_FILEPATH).get();
+        ExportCommand command;
+        switch (directoryPath) {
+        case DEFAULT_DIRECTORY:
```

Comment:
Would it make more sense to have the empty argument `export` command input be the default directory export instead?
Like `export p/asdasd` will export to folder `asdasd`
`export` will export to default

Reply:
Makes sense is makes sense. But what if the user wanted to export to a certain directory and then accidentally press enter after typing export. He has to spend sometime finding what’s the default path then go delete the folder. Btw I tried the default path. It shows it’s to my root path...